### DISCUSSION TITLE
Forcing login to sync and also changing profile storage location

### DISCUSSION THREAD

**Discussion by Comprox** [mozilla/policy-templates#1002] [Q&A]
1. Is there a way via GPO or other means to force someone to login to a Firefox account when using the browser?
2. Is there a way via GPO or other to change the folder where profiles are stored?

We use redirected folders and this makes Firefox store profiles on our file server (since it uses roaming appdata). We want to move these profiles to local and then force staff to login and use sync so they don't lose settings if their PC dies.

Thanks!

**Comment by mkaply** [comment]
We don't have a way to force a Firefox account to be used.

We don't have a way to move the entire Firefox profiles directory, but you can force the profile to a specific directory with a command lin eflag

https://wiki.mozilla.org/Firefox/CommandLineOptions#-profile_.22profile_path.22

These command line options might be useful

https://wiki.mozilla.org/Firefox/CommandLineOptions#User_profile

**Comment by Comprox** [comment]
Sadly those don't fix our situation. While we can make a default shortcut for staff, they can easily make their own shortcut and not use the command line to select the correct profile. 

I see there is a profiles.ini in roaming appdata. Is there some way to set the default location in there or would future made profiles default back to roaming appdata? Sometimes people make new profiles or old ones become corrupted.

Thanks!

**Comment by mkaply** [comment]
I looked into this and unfortunately we have no way to change the profiles directory.
